src/ResourceManager/PowerBIEmbedded/AzureRM.PowerBIEmbedded.psd1

```diff
@@ -112,8 +120,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Add support for online help
-    - Run Get-Help with the -Online parameter to open the online help in your default Internet browser'
+        ReleaseNotes = '* Add support for PowerBI Embedded Capacity'
```

Comment:
@selasagi please revert this change to the release notes - we will handle updating this section during the release using our tooling